src/mmw/apps/modeling/tasks.py

```diff
@@ -94,10 +98,12 @@ def get_histogram_job_results(self, incoming):
     """
     pixel_width = incoming['pixel_width']
     histogram = histogram_finish(incoming['sjs_job_id'], self.retry)
+    aoi = incoming['aoi']
 
     return {
         'pixel_width': pixel_width,
-        'histogram': histogram
+        'histogram': histogram,
+        'aoi': aoi
```

Comment:
Since `area_of_interest` has been added to `histogram_to_survey`, all changes to `get_histogram_job_results` and `start_histogram_job` can be reverted, as they are no longer needed.